crates/kitsune_p2p/proxy/src/tx2.rs

```diff
@@ -1,5 +1,4 @@
 #![allow(clippy::new_ret_no_self)]
-#![allow(clippy::blocks_in_if_conditions)]
```

Comment:
This has been renamed and made more strict, decided to just drop the allow and fix occurrences because it's a valid lint IMO